java/src/jmri/jmrit/operations/setup/EditManifestHeaderTextPanel.java

```diff
@@ -184,7 +183,7 @@ public EditManifestHeaderTextPanel() {
         pManifest.add(pPickup_Comment_TextField);
 
         JPanel pRWE_TextField = new JPanel();
-        pRWE_TextField.setBorder(BorderFactory.createTitledBorder(Bundle.getMessage("RWE")));
+        pRWE_TextField.setBorder(BorderFactory.createTitledBorder(Bundle.getMessage("RWELabel")));
```

Comment:
I always try to avoid using a label string ending in ":" for a border title, so maybe the key named BorderLayoutReturnWhenEmpty from the ops.Cars.bundle would be better suited (and more informative to users guessing what RWE means)?
We might even merge that one with ReturnWhenEmpty, as it reads the same and only confused me when I was looking why my German RWL translations wasn't showing. Just a suggestion.